src/components/molecules/Dropdown/Dropdown.tsx

```diff
@@ -5,7 +5,12 @@ import "react-dropdown/style.css";
 import { getFontFromTheme } from "../../../theme/provider";
 import chroma from "chroma-js";
 
-const StyledDropdown = styled("div")`
+export interface DropdownProps extends ReactDropdownProps {
+  direction?: "up" | "down";
```

Comment:
`direction` most of the time in attributes is used in `rtl` or `lrt` context 

this is called `placement` and not sure if this should be done automatically (calculated by `y`, `height`, etc`) 

<img width="641" alt="image" src="https://user-images.githubusercontent.com/214608/174569054-4a86e7ab-cd20-4e90-8198-24664fa140a4.png">

https://react-select.com/props#select-props

Reply:
good suggestion with naming 👍 

the dropdown behavior should be identical to the tooltip, i.e. when there is not enough space, it sets an alternative placement by itself, the plugin we are currently using does not have it https://www.npmjs.com/package/react-dropdown

Reply:
just change name in this PR